### DISCUSSION TITLE
Quick way to get moves count per team?

### DISCUSSION THREAD

**Discussion by ButteredGroove** [cwendt94/espn-api#334] [ESPN API] [answered]
In the Fantasy Football final standings page, it lists the moves made per team.

Looking at the [Team Class](https://github.com/cwendt94/espn-api/wiki/Team-Class), I don't see a field for it (though there is the mysterious `.mov` field).

Is there a way of quickly getting the moves made count per team?

I believe that I can calculate them using a large size call to the [League function](https://github.com/cwendt94/espn-api/wiki/League-Class#functions) `recent_activity()`.  But it's very slow and then I need to do the counting.

Thank you!

**Comment by ButteredGroove** [comment, marked as answer]
After doing a little investigating, the data I'm looking for lives in the team section under "transactionCounter".  Currently, team.py does not store this data.  Some interesting fields are:
* acquisitionBudgetSpent
* acquisitions
* trades
* paid
* teamCharges
* moveToIR

Thus, this isn't available currently through the API.  I'll create a feature request.

**Reply by cwendt94** [reply]
Added in package v0.19.4

**Comment by cwendt94** [comment]
This is great, thanks for looking into this and finding out some more data that would be perfect under the team class
